server.py

```diff
@@ -1196,9 +1229,14 @@ def put(self):
             uperm = args['new_user_permissions']
             uregions = args['new_user_regions']
 
+            perm = None
+            if uperm == 'Super Admin': perm = M.AdminLevels.SUPER
```

Comment:
for consistency, let's make the values of the permissions on the frontend match the string values of the enums on the backends (i.e. 'SUPER' and 'REGION')